TheKeyOAuth2/TheKeyOAuth2Client.m

```diff
@@ -14,15 +14,15 @@
 
 /* TheKey OAuth2 Settings */
 static NSString *const TheKeyOAuth2ServiceProvider = @"TheKey";
-static NSString *const TheKeyOAuth2RedirectURI     = @"thekey:/oauth/mobile/ios";
-static NSString *const TheKeyOAuth2Scope           = @"fullticket";
+static NSString *const TheKeyOAuth2RedirectURI     = @"/oauth/client/public";
+static NSString *const TheKeyOAuth2Scope           = @"fullticket,extended";
```

Comment:
this might actually need to be `fullticket extended`, which might be causing your ticket issue